mozilla_version/test/test_gecko.py

```diff
@@ -287,6 +287,79 @@ def test_gecko_version_implements_str_operator(version_string, expected_output):
     assert str(GeckoVersion.parse(version_string)) == expected_output
 
 
+@pytest.mark.parametrize('version_string, field, expected', (
+    ('0.9', 'major_number', '1.0'),
+    ('0.9.1', 'major_number', '1.0'),
+    ('1.0b1', 'beta_number', '1.0b2'),
+    ('1.0rc1', 'release_candidate_number', '1.0rc2'),
+    ('1.0', 'patch_number', '1.0.1'),
+    ('1.5', 'minor_number', '1.6'),
+    ('1.5.0.1', 'minor_number', '1.6'),
+    ('1.5', 'old_fourth_number', '1.5.0.1'),
+    ('1.5.0.1', 'old_fourth_number', '1.5.0.2'),
+    ('2.0rc1', 'release_candidate_number', '2.0rc2'),
+    ('2.0', 'old_fourth_number', '2.0.0.1'),
+    ('2.0.0.1', 'old_fourth_number', '2.0.0.2'),
```

Comment:
I can't stop smiling seeing this fourth version. I almost wish we'd ship a `70.0.0.1` these days :P